rb/spec/tests.bzl

```diff
@@ -32,6 +33,29 @@ BROWSERS = {
             "//conditions:default": {},
         }),
     },
+    "chrome-beta": {
+        "data": chrome_beta_data,
+        "deps": ["//rb/lib/selenium/webdriver:chrome"],
+        "tags": [],
+        "target_compatible_with": [],
+        "env": {
+            "WD_REMOTE_BROWSER": "chrome",
+            "WD_SPEC_DRIVER": "chrome-beta",
```

Comment:
we don't want to make these values different, and whatever we do here, we should also do to firefox. I think it might make sense to do `-beta` for all of them and sort it out in test_environment.